docs/index.md

```diff
@@ -40,7 +40,8 @@ curl https://raw.githubusercontent.com/drud/ddev/master/install_ddev.sh | bash
 ```
 
 ### Manual Installation - Linux and macOS
-You can also easily perform the installation manually if preferred:
+
+Alternatively, perform the installation manually.
```

Comment:
I'm pretty sure this should be "Alternately", not "Alternatively". There are probably other places, I think you mentioned that in your basic overview and I didn't catch it.

Reply:
I don't know how I've been on this planet for 37 years and never realized there was an alternative to alternative! http://grammarist.com/usage/alternately-vs-alternatively/. That said, alternative feels right because it's a choice in parallel with the package installers. But I'm not an expert on alternately, so I'm happy to be wrong in this case.

Reply:
I'd have linked the same explanation, heh. 

On second reading, I don't think this is needed at all. I was trying to remove the phrasing "you can X if preferred." 

I'm going to elminate it. 

There's no similar line for the Windows manual install.